src/components/scroll-to.cjsx

```diff
@@ -0,0 +1,65 @@
+React = require 'react'
+_ = require 'underscore'
+
+# Note that the Position mixin methods are called directly rather than mixing it in
+# since we're a mixin ourselves our consumers also include GetPosition and it causes
+# duplicate method name errors to mix it in
+Position = require './get-position-mixin'
+
+DEFAULT_DURATION   = 750 # milliseconds
+DEFAULT_TOP_OFFSET = 80  # pixels
```

Comment:
I wasn't suggesting sharing vars, just having a common string to grep for if we change the CSS to be something other than `69px`